editor/assets/stylesheets/main.scss

```diff
@@ -48,4 +48,25 @@ body.toplevel_page_gutenberg {
 	@include break-small() {
 		padding-top: $header-height;
 	}
+
+	input, textarea {
```

Comment:
This style is overriding the Text mode textarea styles, eliminating padding and changing font appearance.